app/assets/javascripts/tylium/modules/sidebar.js

```diff
@@ -15,10 +16,24 @@
       var that = this;
 
       $('[data-behavior~=sidebar-toggle]').on('click', function(e) {
+        if ($(this).is('[data-behavior~=nodes-tree]')) {
+          if (window.innerWidth < that.minBreakpoint) {
+            $(this).attr('data-behavior', 'nodes-tree sidebar-toggle');
+          } else {
+            $(this).attr('data-behavior', 'nodes-tree sidebar-toggle open-only');
+          }
+        }
+
         if (!(that.isSidebarOpen() && $(this).is('[data-behavior~=open-only]'))) {
           that.toggle(!that.isSidebarOpen());
         }
       });
+
+      if (window.innerWidth < that.minBreakpoint) {
+        $('[data-behavior~=sidebar-node-link]').on('click', function() {
+          that.close();
+        });
+      }
```

Comment:
This hides the nodes menu when a node is clicked so it doesn't stay open and obstruct the view after navigating to a node